django-rgd/setup.py

```diff
@@ -50,7 +50,7 @@
     install_requires=[
         'boto3',
         'celery',
-        'django>=3.2',  # See PR #264: due to this bug: https://code.djangoproject.com/ticket/31910
+        'django==3.2',  # See PR #264: due to this bug: https://code.djangoproject.com/ticket/31910
```

Comment:
Do you want this to be `~=3.2` to allow future versions of `3.x`?

Reply:
I don't think there will be any more minor releases now that Django 4 is out. This should still allow patch updates though.

Also, this is temporary, hopefully only for the next few days (famous last words), until we can support Django 4

Reply:
Changed it though to be fancy with `~=` since I just learned about that yesterday in https://github.com/conda-forge/staged-recipes/pull/17167#discussion_r762512508